.github/workflows/lint.yml

```diff
@@ -29,12 +29,12 @@ jobs:
 
   # hadolint lints the Dockerfile
   hadolint:
-    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_lint.yml@b0827dc668654a431870cf1569552b5429a4b61d #v0.6.0
```

Comment:
multiple CI workflows did not start. could it be because of this line?

Reply:
Thanks fixed. If CI passes, we're good to go